hbase-client/src/main/java/org/apache/hadoop/hbase/shaded/protobuf/ProtobufUtil.java

```diff
@@ -2836,10 +2836,12 @@ public static void mergeFrom(Message.Builder builder, CodedInputStream codedInpu
 
   public static ReplicationLoadSink toReplicationLoadSink(
       ClusterStatusProtos.ReplicationLoadSink rls) {
-    return new ReplicationLoadSink(rls.getAgeOfLastAppliedOp(),
```

Comment:
Have you replaced all?  only here?

Reply:
I checked the code and changed the constructor function to private. There is no problem with compiling. Here is the only invoke.

Reply:
what about codes in unit tests.

Reply:
No explicit invoke in UT. But I ran all the UTs in the same module. They all passed.